services/object/src/api.js

```diff
@@ -10,4 +11,25 @@ let builder = new APIBuilder({
   context: ['cfg', 'db', 'backends'],
 });
 
+builder.declare({
+  method: 'post',
+  route: '/upload/:name/:projectId',
+  name: 'uploadObject',
+  input: undefined,
+  output: undefined,
```

Comment:
I think we'll want to return something that will tell the caller how to actually upload the data.  Since there are no actual backends yet, make something up, using https://docs.taskcluster.net/docs/reference/platform/queue/api#createArtifact for inspiration. :)